packages/template-retail-react-app/app/components/hero/index.jsx

```diff
@@ -52,14 +53,7 @@ const Hero = ({title, img, actions, ...props}) => {
                     paddingTop={{base: 4, lg: 0}}
                 >
                     <Box position={'relative'} width={{base: 'full', md: '80%', lg: 'full'}}>
-                        <Image
-                            fit={'cover'}
-                            align={'center'}
-                            width={'100%'}
-                            height={'100%'}
-                            src={src}
-                            alt={alt}
-                        />
+                        <Image {...imageProps} />
```

Comment:
The hero image is missing the `loading="eager"` attribute. Is that intentional?

Reply:
The eager-loading is passed down to the hero from the home page component. That way not ALL hero images are eagerly loading their images, only the ones that explicitly define it.